### PR TITLE
feat(device): Add tablet detection to device

### PR DESCRIPTION
**Pull request by TNuzzi**
This is a port of `xo-gears` isTablet.  This is needed to detect table on pre-render for Venmo.

### CODE REVIEW

File: src/device.js [hunk 1]
```diff
@@ -13,6 +15,14 @@ export function isDevice(userAgent? : string = getUserAgent()) : boolean {
     return false;
 }
 
+export function isTablet(userAgent? : string = getUserAgent()) : boolean {
```
Comment (gregjopa) [inline src/device.js:18]: Change looks good to me. Could you add some unit tests for this as well? Similar to: https://github.com/krakenjs/belter/blob/master/test/tests/device/isDevice.js

Comment (TNuzzi) [reply src/device.js:18]: @gregjopa It looks like the tests were split into separate files.  I have included test in `test/tests/device/isTablet.js`

Comment (TNuzzi) [reply src/device.js:18]: https://github.com/krakenjs/belter/pull/63/files#diff-460a3c38bb5195e525ae7447c5e2f5a20aca7705defc2c6f4342a7860bef55d6

Comment (gregjopa) [reply src/device.js:18]: Thanks for adding tests @TNuzzi! 💯

**Pull request merged**